great_expectations/render/renderer/notebook_renderer.py

```diff
@@ -159,12 +159,26 @@ def add_expectation_cells_from_suite(self, expectations):
 
             for exp in expectations:
                 kwargs_string = self._build_kwargs_string(exp)
-                meta_args = ", meta={}".format(exp.meta) if exp.meta else ""
+                meta_args = self._build_meta_arguments(exp.meta)
                 code = "batch.{}({}{})".format(
                     exp["expectation_type"], kwargs_string, meta_args
                 )
                 self.add_code_cell(code, lint=True)
 
+    @staticmethod
+    def _build_meta_arguments(meta):
```

Comment:
Just to confirm - we are not just hiding this metadata from the viewer of the notebook, we are actually removing it. Once the suite is saved, this attribute is no longer appears in the JSON file. Is this the intended behavior?

Reply:
This is correct. The notebook can idempotently re-create a suite. Unless we want to venture into magic territory where we store state outside the users' view, this is the best solution.